demo-linux/demo.cpp

```diff
@@ -67,6 +71,26 @@ void loadEEPROM()
     free(values);
 }
 
+int writeToEEPROM()
+{
+    unsigned char buf[64];
+    string writeString;
+    wp_get_eeprom_page(specIndex, 4, buf, 64);
+    printf("The value at page 4 is %s\n", buf);
+    std::stringstream s;
+    s << "Test string. ";
+    s << time(NULL);
+    writeString = s.str();
+    strcpy((char*)buf, writeString.c_str());
+    printf("wrote string to buffer, buffer is now %s\n", buf);
+    int i = wp_write_eeprom_page(specIndex, 4, buf, 64);
+    strcpy((char*)buf, "");
```

Comment:
As alternatives, "buf[0] = 0" would be sufficient, and "memset(buf, 0, 64)" would be thorough :-)